config/urls.py

```diff
@@ -14,8 +14,26 @@
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
 from django.conf.urls import url
+from django.conf.urls.static import static
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
+
+from askup import views
+
+app_name = 'askup'
+
+STATIC_URL = '/assets/'
 
 urlpatterns = [
+    url(r'^$', views.OrganizationsView.as_view(), name='organizations'),
     url(r'^admin/', admin.site.urls),
-]
+    url(r'^login/$', auth_views.login, name='login'),
+    url(r'^logout/$', auth_views.logout, name='logout'),
+    url(r'^org/(?P<pk>[0-9]+)/$', views.OrganizationView.as_view(), name='organization'),
```

Comment:
I would move askup related urls into urls.py module in askup dir and include them in config url